modules/cluster/variables.tf

```diff
@@ -189,6 +189,30 @@ variable "critical_addons_node_group_bottlerocket_admin_container_source" {
   description = "URI of a custom admin container image"
 }
 
+variable "critical_addons_vpc-cni_configuration_values" {
```

Comment:
Possibly contentious variable naming - I've preserved the addon name as-is, including the kebab case. _Possibly_ makes it easier to identify that `vpc-cni` is the addon, and `configuration_values` is the argument.

Happy to go full snake case.

Reply:
On the whole our rule is to just snake case everything for consistency ... 🤷 I am 50/50 on this one though!